scripts/items/ccb_polymer_pump.lua

```diff
@@ -0,0 +1,29 @@
+-----------------------------------------
+-- ID: 5268
+-- Item: CCB Polymer Pump
+-- Used during CoP 6-4 One To Be Feared on Ultima or Omega to inflict a 1 minute amnesia
+-----------------------------------------
+local itemObject = {}
+
+itemObject.onItemCheck = function(target, player)
+    local result      = xi.msg.basic.ITEM_UNABLE_TO_USE
+    local battlefield = player:getBattlefieldID()
+
+    if
+        battlefield == 992 and
```

Comment:
Is this possible to not be hardcoded and use an enum?

Reply:
Funny you mention this, I was just looking at it and yes. 
`        battlefield == xi.battlefield.id.ONE_TO_BE_FEARED and`